elt-pipelines/facility_ops/ingest/computing/jira/jira.py

```
@@ -0,0 +1,109 @@
+            ("all_jira_issues", ResourceProperties(extractor=self.extract_jira_issues)),
+        )
+
+
+def _value_or_env_variable(value: str | None, env_var_name: str) -> str:
+    if value is not None:
+        return value
+    else:
+        try:
+            return os.environ[env_var_name]
+        except KeyError:
+            raise KeyError(f"Environment variable '{env_var_name}' not found.")
```

Ruff (0.16.2)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)